src/mpyl/steps/deploy/k8s/chart.py

```diff
@@ -593,12 +593,14 @@ def to_role_binding(self) -> V1RoleBinding:
             ],
         )
 
-    def to_sealed_secrets(self) -> V1SealedSecret:
+    def to_sealed_secrets(self, release_name: Optional[str] = None) -> V1SealedSecret:
         secrets: dict[str, str] = {}
         for secret in self.sealed_secrets:
             secrets[secret.key] = secret.get_value(self.target)
 
-        return V1SealedSecret(name=self.release_name, secrets=secrets)
+        return V1SealedSecret(
+            name=release_name if release_name else self.release_name, secrets=secrets
```

Comment:
this seems a bit awkward to me, that the `release_name` class prop is not the actual release name?

Reply:
i agree, alternatively i could alter the release name by accessing `sealed_secret.metadata.name` in the `DeployDagster`-step.

Reply:
and for context, we havent setup namespace independent config for dagster user-code yet. thats why we deploy user-code into a singular namespace where everything dagster-related (database,daemon) is also deployed in